src/api/lib/tasks/dev.rake

```diff
@@ -235,6 +235,9 @@ namespace :dev do
 
       # Create news
       Rake::Task['dev:news:data'].invoke
+
+      # Create some Reports
+      Rake::Task['dev:reports:data'].invoke
```

Comment:
@danidoni you have to call this job before line 219, or call the `SendEventEmailsJob` here as well, otherwise we not receiving the notifications.

Reply:
Aaah that's why!

Reply:
Fixed